account_payment_order/models/account_payment_order.py

```diff
@@ -514,3 +514,20 @@ def generate_move(self):
         trfmoves = self._prepare_trf_moves()
         for hashcode, blines in trfmoves.items():
             self._create_reconcile_move(hashcode, blines)
+
+    @api.multi
+    def _all_lines_reconciled(self):
+        """
+        Return true if all payment lines' counterpart lines are reconciled too
+        Also return true if the counterpart account cannot be reconciled
+        """
+        result = True
```

Comment:
Slight preference for `self.ensure_one()` here. It will likely be a mistake when this is called on multiple orders ever.